### PR TITLE
Added correct python shebangs and updated README.md with instructions for Arch Linux'

### PR DESCRIPTION
**Pull request by xflakes**
I have added shebangs to the python scripts, as well as updated the requirements for Arch Linux.

### CODE REVIEW

**Review by rkrp — CHANGES_REQUESTED** [review]
[deleted]

File: README.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 
 ## Requirements
-- System requirements
-
-    `apt-get install qt5-default libqt5webkit5-dev build-essential python-lxml       python-pip xvfb` 
+- System requirements  
+    - Ubuntu  
+        `apt-get install qt5-default libqt5webkit5-dev build-essential python-lxml       python-pip xvfb` 
+    - Arch Linux  
+        `pacman -S install qt5-base qt5-webkit base-devel python2-pip xorg-server-xvfb jre8-openjdk`
```
Comment (rkrp) [inline README.md]: You have included `install` as one of the packages for installation. `pacman` would throw an error failing to find package `install`.

```
error: target not found: install
```

**Review by rkrp — CHANGES_REQUESTED** [review]
Arch installation instruction is incorrect.

**Comment by xflakes** [comment]
Not sure how that got in there, sorry.
Removed the 'install'.

**Comment by pwnfoo** [comment]
@flhp Please clone from the upstream for sending in PR's. Accepting this PR is only going to pollute the history. Also, please send in your PR's from a separate branch. Closing this as this in invalid.

**Pull request closed without merging**